README.md

```diff
@@ -152,6 +144,26 @@ type definitions:
 - Rust: https://github.com/package-url/packageurl.rs
 - Swift: https://github.com/package-url/packageurl-swift
 
-## Users, adopters and links
+## Support
+
+If you have a specific problem, suggestion or bug, please submit a 
+[GitHub issue](https://github.com/package-url/purl-spec/issues).
+
+For quick questions or socializing, join the PURL community discussions 
+at:
+- [AboutCode Slack](https://aboutcode-org.slack.com/archives/C08LBQMA7DE)
+- [CycloneDX Slack](https://cyclonedx.slack.com/archives/C06KTE3BWEB)
+- [Gitter](https://matrix.to/#/#package-url_Lobby:gitter.im)
+
+## License
+
+Copyright (c) the purl authors
+
+The license for **purl-spec** software is MIT: 
+- SPDX-License-Identifier: MIT-License.
+- See the *LICENSE* file in this repository for the license text.
```

Comment:
This will be detected by ScanCode and is a source of ambiguity, so this is best avoided.

The file named `LICENSE` should be enough.

```suggestion
```